MobileCenterCrashes/MobileCenterCrashes/Internals/MSWrapperCrashesHelper.h

```diff
@@ -17,4 +20,24 @@
  */
 + (id<MSCrashHandlerSetupDelegate>)getCrashHandlerSetupDelegate;
 
+/**
+ * Enables or disables automatic crash processing. Setting to 'NO' causes SDK not to send reports immediately, even if ALWAYS_SEND is set.
+ */
++ (void)setAutomaticProcessing:(BOOL)automaticProcessing;
+
+/**
+ * Gets a list of unprocessed crash reports.
+ */
++ (NSArray<MSErrorReport *> *)getUnprocessedCrashReports;
```

Comment:
Apple naming convention recommend not to prepend `get` to any method returning a value except when it's none obvious for instance when objects and values are returned indirectly. The [Apple doc](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-1004202-BCIGGFCC) explains it in the accessor section but it tend to be a general rule.

Reply:
What name do you recommend?

Reply:
just remove get?